### PR TITLE
[FE] Feat thinking comment

### PR DESCRIPTION
**Pull request by maishet**
The following changes have been made:

- New post component created.
- The profile icon now has a light blue background.
- Privacy button added.
- A field was added to place a description of the publication.

image

### CODE REVIEW

File: frontend/src/app/test/page.tsx [hunk 1]
```diff
@@ -110,6 +110,51 @@ export default function TestPage() {
...
       <AvatarInput avatarValue={avatar} onChangeAvatar={(avatar)=>setAvatar(avatar)} />
+      
+      <div
+  style={{
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+    background: "#fff",
+    padding: "20px",
+    boxShadow: "0 4px 8px rgba(0, 0, 0, 0.1)",
```
Comment (Anthonyricse) [inline frontend/src/app/test/page.tsx:121]: create component in carpet molecule 

example commentThink  
use material iu : https://mui.com/joy-ui/react-textarea/   text area

Comment (Anthonyricse) [reply frontend/src/app/test/page.tsx:121]: example AvatarInput 
image

Comment (maishet) [reply frontend/src/app/test/page.tsx]: The component was moved to the molecules folder and the design presented in Figma was followed.

**Review by AnthonyRicse98 — APPROVED** [review]

**Comment by maishet** [comment]
## Summary
The code snippet is a part of the `CommentThink` function in a TypeScript React component. It receives three props (`avatarDefaultURL`, `publicTag`, and `placeholder`) and returns a JSX element.

## Example Usage
```javascript
<CommentThink avatarDefaultURL="url" publicTag="tag" placeholder="Enter your comment" />
```

## Code Analysis
### Inputs
- `avatarDefaultURL` (optional): a string representing the URL of the default avatar image.
- `publicTag` (optional): a string representing the public tag.
- `placeholder` (optional): a string representing the placeholder text for the textarea.
___
### Flow
The `CommentThink` function receives three props: `avatarDefaultURL`, `publicTag`, and `placeholder`. It returns a JSX element that represents a comment input section. Inside the JSX element, there is a container div with the class name "comment_think_main_container". Inside this container, there is another div with the class name "comment_think_container". Inside this second div, there is a `WCircleImage` component and a `WButton` component. Below the container div, there is a `Textarea` component. The `Textarea` component has various props set, such as style, color, minRows, and placeholder.
___
### Outputs
The output of the code snippet is a JSX element representing a comment input section.
___

**Pull request merged**